src/cluster.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import * as MapboxGL from 'mapbox-gl';
```

Comment:
This thing will totally destroy tree-shaking. Same goes for `* as React`. Don't do that in libraries ;)